examples/index.html

```diff
@@ -33,13 +35,13 @@
         <!-- Nav tabs -->
         <div class="sidebar-tabs">
             <ul role="tablist">
-                <li><a href="#home" role="tab"><i class="fa fa-bars"></i></a></li>
+                <li><a href="#home" role="tab"><i class="fa fa-bars active"></i></a></li>
                 <li><a href="#profile" role="tab"><i class="fa fa-user"></i></a></li>
                 <li class="disabled"><a href="#messages" role="tab"><i class="fa fa-envelope"></i></a></li>
             </ul>
 
             <ul role="tablist">
-                <li><a href="#settings" role="tab"><i class="fa fa-gear"></i></a></li>
+                <li><a href="//github.com/nickpeihl/leaflet-sidebar-v2"><i class="fa fa-github"></i></a></li>
```

Comment:
I think protocol relative URLs is an anti-pattern. Can we just use https://? [More Info](https://www.paulirish.com/2010/the-protocol-relative-url/)

Reply:
:+1: